Mage/Task/BuiltIn/Releases/RollbackTask.php

```diff
@@ -87,6 +87,8 @@ public function run()
                         $releaseIsAvailable = true;
                     }
                 }
+                
+                $currentCopy = $releasesDirectory . '/' . $releaseId;
```

Comment:
I think it'd be more secure if `$releases` would be `rtim`-med before appending `$releaseId`:
```php
$currentCopy = rtrim($releasesDirectory, '/') . '/' . $releaseId;
```